jenkins/automation/vra_infra_provisioner.groovy

```diff
@@ -0,0 +1,100 @@
+pipeline {
+    agent {
+        node {
+            label 'terraform-node'
+        }
+    }
+    
+    options {
+        timeout(time: 30, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '30', numToKeepStr: '30'))
+        ansiColor('xterm')
+    }
+
+
+    parameters {
```

Comment:
Please add a parameter to take `vra_catalog_item` also as an option. So that other teams can use this Jenkins pipeline.

Reply:
Done